tests/manage/z_cluster/nodes/test_non_ocs_taint_and_toleration.py

```diff
@@ -159,36 +168,37 @@ def test_non_ocs_taint_and_tolerations(self):
             logger.info(f"Successfully added toleration to {configmap_obj.kind}")
 
         if config.ENV_DATA["mcg_only_deployment"]:
-            # Wait some time after adding toleration for pods respin.
+            logger.info("Wait some time after adding toleration for pods respin")
             waiting_time = 60
             logger.info(f"Waiting {waiting_time} seconds...")
             time.sleep(waiting_time)
-
-            # Force delete all pods.
+            logger.info("Force delete all pods")
             pod_list = get_all_pods(
                 namespace=config.ENV_DATA["cluster_namespace"],
                 exclude_selector=True,
             )
             for pod in pod_list:
                 pod.delete(wait=False)
 
-        # After edit noticed few pod respins as expected
+        logger.info("After edit noticed few pod respins as expected")
         assert wait_for_pods_to_be_running(timeout=600, sleep=15)
 
-        # Check non ocs toleration on all pods under openshift-storage
+        logger.info("Check non-ocs toleration on all pods under openshift-storage")
         check_toleration_on_pods(toleration_key="xyz")
         self.sanity_helpers.health_check()
 
-        # check number of pods before and after adding non ocs taint
+        logger.info("Check number of pods before and after adding non ocs taint")
         number_of_pods_after = len(
             get_all_pods(namespace=config.ENV_DATA["cluster_namespace"])
         )
         assert (
             number_of_pods_before == number_of_pods_after
         ), "Number of pods didn't match"
 
-        # Add capacity to check if new osds has toleration
-        if not config.ENV_DATA["mcg_only_deployment"]:
+        logger.info("Add capacity to check if new osds has toleration")
```

Comment:
There is no need to log this message on the external mode cluster as we don't do it. Adding it here can create confusion. Please add this log information after the if condition.

Reply:
+1

Reply:
Right, done